src/components/GroupedTaskStatusBadge/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Body } from "@leafygreen-ui/typography";
+import { taskStatusToCopy } from "constants/task";
+import { TaskStatus } from "types/task";
+
+interface Props {
+  status: TaskStatus;
+  count: number;
+}
+export const GroupedTaskStatusBadge: React.FC<Props> = ({ count, status }) => {
+  let statusDisplayName = taskStatusToCopy[status];
+  if (statusDisplayName.slice(-1) === "s" && count !== 1) {
+    statusDisplayName += "es";
+  } else if (statusDisplayName.slice(-1) === "e" && count !== 1) {
+    statusDisplayName += "s";
+  }
+
+  return (
+    <BadgeContainer css={statusToColorVariant[status]}>
+      <Number css={statusToColorVariant[status]}>{count}</Number>
+      <Status css={statusToColorVariant[status]}>{statusDisplayName}</Status>
+    </BadgeContainer>
+  );
+};
+
+const BadgeContainer = styled.div`
+  height: 27px;
+  width: 57px;
+  border-radius: 3px;
+  border: 0.75px solid;
+  box-sizing: border-box;
+  border-radius: 3px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+`;
+
+const Number = styled(Body)`
+  font-weight: bold;
+  font-size: 11px;
+  letter-spacing: 0.18px;
+  line-height: 10px;
+`;
+
+const Status = styled(Body)`
+  font-size: 8px;
+  letter-spacing: 0.13px;
+  line-height: 10px;
+`;
```

Comment:
Is this styling necessary? I would like to avoid magic numbers and would prefer if we were more consistent/deliberate about the styles we apply. 
Could we update some of these values to more closely follow the leafygreen guidelines.  https://www.mongodb.design/component/typography/guidelines/ 
Specifically surrounding the typography elements.

Reply:
For `Number` and `Status`, I think line-height is necessary because otherwise the words will be out of the box like this:
<img width="103" alt="Screen Shot 2021-07-30 at 5 37 32 PM" src="https://user-images.githubusercontent.com/55669681/127714307-03d34980-56b8-441e-8b7f-1f252e13ad67.png">

Same thing applies to font-size, they would spell outside the box.
Other stylings were also copied from Figma, but I think we can removed the letter-spacing.

Reply:
I think Mohamed makes a good point here! One approach could be to not use LG's `Body` component at all in favor of a more neutral `<p>` or `<span>` tag. IMO this badge doesn't really represent body text, as it has a pretty different purpose and style from a normal paragraph with text, which is what `Body` is designed for.